lib/google_maps_geocoder/google_maps_geocoder.rb

```diff
@@ -3,6 +3,8 @@
 require 'net/http'
 require 'rack'
 
+require 'google_maps_geocoder/google_maps_geocoder_error_handler'
```

Comment:
Let's rename `google_maps_geocoder_error_handler` to `geocoding_exception`. Shorter and sweeter.